src/components/textarea/textarea.mdx

```diff
@@ -170,6 +170,11 @@ It is possible to use the `tooltipPosition` to override the default placement of
 
 **Any other supplied props will be provided to the underlying HTML input element**
 
+## Translation keys
+
+The following keys are available to override the translations for this component by passing in a custom locale object to the
+[i18nProvider](https://carbon.sage.com/?path=/story/documentation-i18n--page).
```

Comment:
**suggestion**:  due to the Storybook v7 upgrade, some of the URLs generated are now different so this link would break.

Would you be able to use the following URL @tomdavies73?

```suggestion
[i18nProvider](../?path=/docs/documentation-i18n--docs).
```